bedwars-plugin/src/main/java/com/andrei1058/bedwars/BedWars.java

```diff
@@ -320,7 +319,7 @@ public void onEnable() {
                 //registerEvents(new ArenaListeners());
                 ArenaSocket.lobbies.addAll(config.getList(ConfigPath.GENERAL_CONFIGURATION_BUNGEE_OPTION_LOBBY_SERVERS));
                 new SendTask();
-                registerEvents(new AutoscaleListener(), new PrePartyListener(), new JoinListenerBungee());
+                registerEvents(new AutoscaleListener(), new JoinListenerBungee());
```

Comment:
Whats the reason of removing the PrePartListener?

Reply:
It didnt do anything, its just called it for nothing and it was deprecated with only 2 usages (when listening to it)

Reply:
I see it was marked for removal aswell